.github/workflows/lambda-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+
+on: push
+
+jobs:
+  test_lambdas:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Test lambdas
+        working-directory: ./server/aws/lambda
+        run: |
+          npm install
```

Comment:
Should be `npm ci` so we don't accidentally upgrade package versions in CI 

https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
Nice catch